internal/core/command/put_test.go

```diff
@@ -26,18 +28,21 @@ import (
 func TestNewPutCommandWithCorrelationId(t *testing.T) {
 	expectedCorrelationIDHeaderValue := "Testing"
 	testContext := context.WithValue(context.Background(), clients.CorrelationHeader, expectedCorrelationIDHeaderValue)
-	putCommand, _ := NewPutCommand(testDevice, testCommand, "Test body", testContext, nil, logger.NewMockClient())
+	req := newRequestForPutWithContentType(clients.ContentTypeCBOR)
+	putCommand, _ := NewPutCommand(testDevice, testCommand, "Test body", testContext, nil, logger.NewMockClient(), req)
 	actualCorrelationIDHeaderValue := putCommand.(serviceCommand).Request.Header.Get(clients.CorrelationHeader)
 	if actualCorrelationIDHeaderValue == "" {
-		t.Errorf("The populated PutCommand's request should contain a correlation ID header value")
```

Comment:
Can we revert this change since this is less informative then it was previously? This goes for the other changes to error messages that change `request` to `req`.